src/components/SQForm/SQFormMultiSelect.js

```diff
@@ -31,16 +31,20 @@ const MenuProps = {
   getContentAnchorEl: null
 };
 
+const selectedDisplayValue = (values, options) => {
+  return values
+    .map(value => {
+      return options.find(option => option.value === value).label;
```

Comment:
Looking at the prop-type it appears `options` could possibly be `undefined`

```javascript
  /** Multiselect options to select from */
  children: PropTypes.arrayOf(
    PropTypes.shape({
      label: PropTypes.string,
      value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
    })
  ),
```

Can we add in optional chaining to guard against `options`, `option.value`, and `option.label` potentially being `undefined`?

Reply:
This brings up another question: should we allow children to be `undefined`? I think `children`, `option`, `option.label` and `option.value` should be required. Otherwise it's possible to have a select with no options. is that a usecase we want/care about?

Reply:
Okay, I'll test some undefined values and see if I can figure out where it may fail